Formula/cling.rb

```diff
@@ -0,0 +1,38 @@
+# Documentation: https://docs.brew.sh/Formula-Cookbook.html
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```

Comment:
I am sure you will understand that reading in your pull request the comment saying `+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!` does nothing to persuade us that the formula will follow Homebrew's rules for new formulas…